packages/account/src/account.ts

```diff
@@ -642,6 +646,22 @@ export class Account extends AbstractAccount {
     return this.provider.simulate(transactionRequest, { estimateTxDependencies: false });
   }
 
+  /**
+   * Generates an array of fake resources based on the provided coins.
+   *
```

Comment:
We should probably indicate the use case of this method
```suggestion
* This method is useful for unfunded wallets to perform dry runs
```

Reply:
@maschad Thanks for the suggestion but I reckon my description suits the method better.

Reply:
@maschad Thanks for the suggestion but I reckon the current description suits the method better.